terasoluna-gfw-web/src/test/java/org/terasoluna/gfw/web/token/transaction/TransactionTokenIssue364InterceptorTest.java

```diff
@@ -0,0 +1,485 @@
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+import static org.hamcrest.CoreMatchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Matchers.*;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.ui.Model;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import org.springframework.web.method.HandlerMethod;
+import org.terasoluna.gfw.web.token.TokenStringGenerator;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(locations = { "classpath:test-context.xml" })
+public class TransactionTokenIssue364InterceptorTest {
+
+    @Autowired
+    ConfigurableApplicationContext context;
+
+    TransactionTokenInterceptor interceptor;
+
+    MockHttpServletRequest request;
+
+    MockHttpServletResponse response;
+
+    @Before
+    public void setUp() throws Exception {
+
+        // prepare request object
+        request = new MockHttpServletRequest();
+        request.setRequestURI("/token_issue364/first");
+        request.setMethod("GET");
+
+        // prepare response object
+        response = new MockHttpServletResponse();
+
+        // set ServletRequestAttributes to RequestContextHolder
+        ServletRequestAttributes attributes = new ServletRequestAttributes(request);
+        RequestContextHolder.setRequestAttributes(attributes);
+
+        // prepare intercepter instance
+        interceptor = new TransactionTokenInterceptor();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+    }
+
+    @Test
+    public void testPreHandleIncorrectHandler() throws Exception {
+        boolean result = interceptor.preHandle(request, response, null);
+        assertTrue(result);
+    }
+
+    @Test
+    public void testPreHandleWithoutTokenValidate() throws Exception {
+        boolean result = interceptor
+                .preHandle(
+                        request,
+                        response,
+                        new HandlerMethod(new TransactionTokenSampleIssue364Controller(), TransactionTokenSampleIssue364Controller.class
+                                .getDeclaredMethod("first", SampleForm.class,
+                                        Model.class)));
+        assertTrue(result);
+    }
+
+    @Test
+    public void testPreHandleWithTokenValidate() throws Exception {
```

Comment:
These test cases you add are really related to #364?
In my understanding it is enough to add a few cases for using ``namespace`` attribute.